lib/mayaUsd/commands/baseExportCommand.cpp

```diff
@@ -16,287 +16,295 @@
 //
 #include "baseExportCommand.h"
 
-#include <vector>
+#include <mayaUsd/fileio/jobs/jobArgs.h>
+#include <mayaUsd/fileio/utils/writeUtil.h>
 
 #include <pxr/pxr.h>
 
-#include <maya/MGlobal.h>
-#include <maya/MSyntax.h>
 #include <maya/MArgDatabase.h>
 #include <maya/MArgList.h>
 #include <maya/MFileObject.h>
+#include <maya/MGlobal.h>
 #include <maya/MSelectionList.h>
+#include <maya/MSyntax.h>
 
-#include <mayaUsd/fileio/jobs/jobArgs.h>
-#include <mayaUsd/fileio/utils/writeUtil.h>
+#include <vector>
 
 PXR_NAMESPACE_USING_DIRECTIVE
 
-MAYAUSD_NS_DEF {
```

Comment:
MAYAUSD_NS_DEF isn't detected as a namespace, and so clang-format applies an indent to everything inside.

I tested using clang's `NamespaceMacros` option; unfortunately, it didn't help.  (Apparently those have a very narrow use case - namespace macros where the name is given as an arg.)

The only way we can avoid this is if we instead start using something like `PXR_NAMESPACE_OPEN_SCOPE` and `PXR_NAMESPACE_CLOSE_SCOPE` pairs, which also include the braces - ie, `MAYAUSD_NAMESPACE_OPEN_SCOPE` and `MAYAUSD_NAMESPACE_CLOSE_SCOPE`.  (We could keep the `MAYAUSD_NS_DEF` for backwards compatibility with dependent code already using it, but not use it within this repo.)

Would we rather do that, or just be ok with the `MAYAUSD_NS_DEF` contents being indented?

Reply:
This was raised by @luceric-adsk internally. The idea is to change `MAYAUSD_NS_DEF` and take the namespace keyword out of it.